### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by belauzas**
Truksta normalesnio stiliaus, reikia padaryti su atskira uzduotimi

### CODE REVIEW

**Review by SvetaKo — DISMISSED** [review]
ok

**Review by JonasChair — APPROVED** [review]
LGBT

**Comment by Tokenas** [comment]
LGTM

**Review by petraspk — DISMISSED** [review]

File: js/function.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 "use strict";
 
 function generateServices( data ) {
-    var HTML = '';
```
Comment (JonasChair) [inline js/function.js]: Missing ";" (kabliataskis)

**Review by JonasChair — CHANGES_REQUESTED** [review]
Missing checks for elements of the incoming Array.
Make missing elements non visible? / set important emelents

**Review by JonasChair — APPROVED** [review]

**Pull request merged**